test/tesla/adapter/gun_test.exs

```diff
@@ -6,26 +6,27 @@ defmodule Tesla.Adapter.GunTest do
   use Tesla.AdapterCase.Multipart
   use Tesla.AdapterCase.StreamRequestBody
   use Tesla.AdapterCase.SSL
+  alias Tesla.Adapter.Gun
```

Comment:
Seems to be used in single place only, might not be necessary

Reply:
After adding some tests it's used 5 times